jenkins-audit/check-jobs.py

```diff
@@ -44,20 +45,40 @@
 
 github_branches = []
 
-job_dir = "/home/jenkins/jenkins/jobs/"
-def check_recips (recips):
+def check_recips (description, recips):
+    # Jobs should always email someone.
     if recips == None:
         print("Error, job %s has no recipients set" % job)
         return True
+
+    # Most test should email testing@ on failure
+    should_email_testing = True
+    # There is a magic description string to flag tests being written which
+    # should email their creator, rather than everyone.
+    dev_test_string = 'This test is under active development.'
+    if dev_test_string in description.text:
```

Comment:
This line caused `TypeError: argument of type 'NoneType' is not iterable` on some machine e.g. Jenkins master (with python v2.6.9).